src/main/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptor.java

```diff
@@ -293,11 +314,12 @@ public void validateArguments() {
 
     /**
      * Determine if a particular ReadFilter was disabled on the command line.
-     * @param filterName name of the filter to query
-     * @return true if the name appears in the list of disabled filters
+     * @param filterName name of the filter to query.
+     * @return {@code true} if the name appears in the list of disabled filters, or is a tool default not provided by the user and all tool defaults are disabled; {@code false} otherwise.
      */
     public boolean isDisabledFilter(final String filterName) {
-        return disableFilters.contains(filterName);
+        return disableFilters.contains(filterName)
+                || (disableToolDefaultReadFilters && !userReadFilterNames.contains(filterName));
```

Comment:
I'm not sure why this change is needed  ? If it is, then the first line of the javadoc needs to change to reflect what it actually does.